net/acme-acmesh/Makefile

```diff
@@ -51,13 +51,14 @@ endef
 define Package/acme-acmesh-dnsapi
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+acme-acmesh
+  DEPENDS:=+acme-common +PACKAGE_uacme:curl
   TITLE:=DNS API integration for ACME (Letsencrypt) client
   PKGARCH:=all
 endef
 
 define Package/acme-acmesh-dnsapi/description
  This package provides DNS API integration for ACME (Letsencrypt) client.
+ scripts are from acme.sh
```

Comment:
The second line in the description does not conform to English grammar and isn't descriptive enough IMHO. Do you need any help redoing the description for this?

Reply:
I wanted to note that those functions are from acme.sh, as this PR make it can linked by other acme client too. But now not sure if it needed as it's name is acme-acmesh-dnsapi already:
Maby it'd better another package that links to this because uacmes dnsapi adapter got quite big (~30kb)